messages/EventKeyTemplates.js

```diff
@@ -19,6 +19,7 @@ module.exports = {
 		"telemetry": "This message is unused."
 	},
 	"warning": {
+		"pmd7IncompatibleRule": "Please add a `language` property to PMD rule [%s] to ensure compatibility with PMD 7. If this is non-viable for you, please log an issue on github.com/forcedotcom/sfdx-scanner",
```

Comment:
How about:
"Add `language` property to custom PMD rule [%s] to add forward compatibility with PMD7."

Will let @teresa-allen-sfdc make it nice.